.github/workflows/031-ubuntu-aurae-builder-image-ubuntu-docker-build-tester.yml

```diff
@@ -35,7 +35,7 @@ permissions:
  packages: write
 env:
  CARGO_TERM_COLOR: always
- CACHE_VERSION: v1
+ CACHE_VERSION: v2
```

Comment:
Is this an intentional change?

Reply:
Yea. I was working on splitting the dockerfile builds at the same time as this PR, and that PR overwrote the image this run needed. Flipping this forced it to use a new clean image.